ffs/api/api_wallet.go

```diff
@@ -81,14 +80,6 @@ func (i *API) VerifyMessage(ctx context.Context, addr string, message, signature
 	return ok, nil
 }
 
-// SendFil sends fil from a managed address to any another address, returns immediately but funds are sent asynchronously.
```

Comment:
Do we need to delete it from the internal API? Sounds to me that only at the gRPC level is enough.

Reply:
Yea, actually realized that and reversing now. Same in wallet module.

Reply:
doh... this just made me wonder. couldn't it be a flag? seems like those running their own Pow would still want these.

Reply:
hmmmm, I'm not sure. thinking.